plugins/robots/generators/generatorBase/src/rules/simpleRules/simpleIfInsideCycleRule.cpp

```diff
@@ -36,6 +36,11 @@ bool SimpleIfInsideCycleRule::apply()
 
 	LoopNode * const parentLoop = makeLoopStartingFrom(mNextNode);
 
+	/// If this IfNode is simple we don't need to check other branch
```

Comment:
If both branches of parent conditional node lead into the same element then no need to add breaks.